docker/synapse_sytest.sh

```diff
@@ -5,6 +5,13 @@
 
 set -ex
 
+if [ -n "$BUILDKITE" ]
+then
+    SYNAPSE_DIR=`pwd`
```

Comment:
could we not set `workdir` in the buildkite config (https://github.com/buildkite-plugins/docker-buildkite-plugin#workdiroptional-string) which would solve this and make the readme (which says: mount the server to be tested at /src) more truthful?

Reply:
see https://github.com/matrix-org/synapse/pull/5664

Reply:
so we can get rid of this condition now?

Reply:
yes